nomic/atlas.py

```diff
@@ -28,35 +28,33 @@ def map_embeddings(
     build_topic_model: bool = True,
     topic_label_field: str = None,
     num_workers: None = None,
-    organization_name: str = None,
     reset_project_if_exists: bool = False,
     add_datums_if_exists: bool = False,
     shard_size: None = None,
     projection_n_neighbors: int = DEFAULT_PROJECTION_N_NEIGHBORS,
     projection_epochs: int = DEFAULT_PROJECTION_EPOCHS,
     projection_spread: float = DEFAULT_PROJECTION_SPREAD,
-) -> AtlasProject:
+) -> AtlasDataset:
     '''
 
     Args:
         embeddings: An [N,d] numpy array containing the batch of N embeddings to add.
         data: An [N,] element list of dictionaries containing metadata for each embedding.
         id_field: Specify your data unique id field. This field can be up 36 characters in length. If not specified, one will be created for you named `id_`.
-        name: A name for your map.
+        name: A name for your dataset. Specify in the format `organization/project` to create in a specific organization.
         description: A description for your map.
         is_public: Should this embedding map be public? Private maps can only be accessed by members of your organization.
-        colorable_fields: The project fields you want to be able to color by on the map. Must be a subset of the projects fields.
-        organization_name: The name of the organization to create this project under. You must be a member of the organization with appropriate permissions. If not specified, defaults to your user accounts default organization.
-        reset_project_if_exists: If the specified project exists in your organization, reset it by deleting all of its data. This means your uploaded data will not be contextualized with existing data.
-        add_datums_if_exists: If specifying an existing project and you want to add data to it, set this to true.
+        colorable_fields: The dataset fields you want to be able to color by on the map. Must be a subset of the projects fields.
```

Comment:
For pushing out a 3.0 release, we could simply deprecateWarning colorable_fields, with a plan to remove… and then going forward it makes more sense to just deprecateWarning this when passed rather than remove to avoid breaking old code.

Reply:
sounds good